docs/glsl-fuzz-walkthrough.md

```diff
@@ -698,6 +724,101 @@ the reduction of this variant leads to the following files:
   reduction. It typically refers to the smallest shader the reducer could obtain
   for this particular reduction.
 
+
+## Inspecting results for compute shader families
+
+A compute shader takes its inputs via uniforms and a single SSBO, and writes results via the SSBO.  The final state of the SSBO for a compute shader is stored in the `ssbo` sub-field of the `outputs` field of the `.info.json` result file produced upon running the shader.  The `ssbo` subfield is a list of lists: there is one list for each field of the SSBO, and it contains a list of data elements associated with that field.
+
+For instance, suppose a compute shader's SSBO is declared as follows:
+
+```
+layout(std430, binding = 0) buffer doesNotMatter {
+ int result;
+ int data[];
+};
+```
+
+A possible end-state for the SSBO is:
+
+```
+[[88],[28,12,14,14,18,16,18,18,28,22,24,24,28,26,28,28,48,32,34,34,38,36,38,
+38,48,42,44,44,48,46,48,48,68,52,54,54,58,56,58,58,68,62,64,64,68,66,68,68,88,72
+,74,74,78,76,78,78,88,82,84,84,88,86,88,88]]
+```
+
+This corresponds to a value of 88 for the `result` field of the SSBO, and values of 28, 12, 14, etc., for the elements of the `data` array field.
+
+### Showing the SSBO for a compute shader result
+
+To view the SSBO for a compute shader result in pretty-printed form, do:
+
+```sh
+# The 'show' argument specifies that we wish to view the SSBO for the given result file
+inspect_compute_results show /path/to/results/reference_or_variant.info.json
```

Comment:
In the first release we made all command names use hyphens rather than underscore (e.g. `glsl-to-spv-worker`), I suggest we do the same for `inspect_compute_results` -> `inspect-compute-results`

Reply:
I agree, and done (though a bit tricky since another Python file needs to import inspect_compute_results and a hyphen cannot appear in an import).